Makefile.PL

```diff
@@ -2,17 +2,28 @@ use ExtUtils::MakeMaker;
 
 # Use: perl Makefile.PL OPTIMIZE="-O0 -g -Wdeclaration-after-statement"
 
-my $yazconf = "yaz-config";
-my $yazver = `$yazconf --version`;
-my $yazinc = `$yazconf --cflags servers`;
-my $yazlibs = `$yazconf --libs server`;
-if (!$yazver || (!$yazinc && !$yazlibs)) {
-   die qq[
+my $yazver;
+my $yazinc;
+my $yazlibs;
+system("pkg-config --exists yaz-server");
+if ($? == 0) {
+    $yazconf = "pkg-config";
+    $yazver = `pkg-config --modversion yaz-server` or die;
+    $yazinc = `pkg-config --cflags yaz-server` or die;
+    $yazlibs = `pkg-config --libs yaz-server` or die;
+} else {
+    my $yazconf = "yaz-config";
+    $yazver = `$yazconf --version`;
+    $yazinc = `$yazconf --cflags servers`;
+    $yazlibs = `$yazconf --libs server`;
+    if (!$yazver || (!$yazinc && !$yazlibs)) {
+       die qq[
 ERROR: Unable to call script: yaz-config
```

Comment:
"... or pkg-config"

Reply:
Probably not. pkg-config will complain and die.. pkg-config is much more verbose than yaz-config.